integration-tests/package.json

```diff
@@ -11,6 +11,7 @@
     "lint:check": "eslint .",
     "build": "hardhat compile",
     "test:integration": "hardhat --network optimism test",
+    "test:actor": "IS_LIVE_NETWORK=true ts-node actor-tests/lib/runner.ts",
```

Comment:
I don't see `IS_LIVE_NETWORK` consumed anywhere in the files added to this PR

Reply:
It's used by the `OptimismEnv` class.